### DISCUSSION TITLE
AdguardHome and IPv6 - Names of the clients

### DISCUSSION THREAD

**Discussion by nextcloudfan** [AdguardTeam/AdGuardHome#7349] [Q&A]
Hello,

I have AdguardHome running on a Pi in my home network. However, the client names are only displayed correctly for devices with IPv4 only. All IPv6 devices are displayed every day with a different IP in the AdGuard client menu.

Can I somehow force the Fritzbox Router to also assign IPv6 statically? Or is there any helpful setting in Adguard?

I have also added the MAC to the devices in the Adguard client menu, but it doesn't help. And yes, my MAC addresses are NOT dynamic in the home network. They are fixed.

Thank you

**Comment by andbez** [comment]
Hi, 
just read https://gist.github.com/timothyham/dd003dbad5614b425a8325ec820fd785 to understand the different IPv6-Adress-Types and how they work.  If you are using a FritzBox, set the Router Advertisement (in German: Router Advertisement im LAN aktiv und ULA zuweisen.) Also activate DNSv6-Server in LAN (RFC 5006).
Hint: The ULA is a kind of static IP-Addres in IPv6.

**Comment by nextcloudfan** [comment]
I have exactly the settings you mentioned activated. You sure?

**Comment by daniel-w00** [comment]
Does it work for you now?
